src/shared/components/communication-framework/contact-us/contact-us-form/initial/ContactUsInitialForm.tsx

```diff
@@ -223,10 +224,11 @@ const ContactUsInitialForm = () => {
       detail: {
         category: 'contact_us',
         action: 'contact_form_submited'
-      }
+      },
+      bubbles: true
     });
 
-    window.dispatchEvent(trackContactUsSubmission);
+    analyticsRef.current?.dispatchEvent(trackContactUsSubmission);
```

Comment:
We could probably reuse `dropAreaRef` here?

Reply:
I tried but `dropAreaRef` is a function.

Reply:
I pushed a commit to address this.

Reply:
Nice. Thanks.